app/test_main.py

```diff
@@ -1 +1,57 @@
-# write your code here
+from typing import Callable
+from unittest import mock
+
+import pytest
+
+from app.main import can_access_google_page
+
+GOOGLE_URL = "https://www.google.com"
+
+
```

Comment:
Can you try to use fot this task Use @pytest.parametrize
also there should be tests for all cases
True,true
True,False
False,True
False, False